.github/workflows/workflow-markdown-dead-link-check.yml

```diff
@@ -0,0 +1,14 @@
+name: Check Markdown links
+
+on:
+  workflow_call:
+
+jobs:
+  markdown-link-check:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@master
+
+    - name: Validating links
+      uses: gaurav-nelson/github-action-markdown-link-check@v1
```

Comment:
the little red marker is indicating that you're missing an end of file new line. this should also be an automated check and also automatically done by the editor:

https://gist.github.com/camh-/1bebfcff1b0f814e9b191edc60d5206b

Reply:
changed and updated the old files that didn't have one not sure how to automatically do it tho when I google it it's mostly extensions showing and still need to do it manually

Reply:
I'm not really too sure what the red marker signifies I added the EOF line and still there
<img width="310" alt="Screenshot_149" src="https://github.com/ai-cfia/github-workflows/assets/9827730/3ef6a83f-34fe-472e-b01f-28ab1cb288e9">

Reply:
...because you also added empty spaces on that last line...

Reply:
If you could talk to me respectfully it would be appreciated, I am not dumb I skipped a line and the tabulation automatically formatted the line I didn't realise someone instructed me to just press enter to fix the issue

Reply:
where did I talked to you disrespectfully?

Reply:
excessive use of ... makes it seem as if it was an obvious thing which I know it was for you but I just applied the fix this is a new concept for me

Reply:
@redat97 you do not resolve conversations initiated by the reviewers, it's up to the reviewer to resolve conversations after verifying the fixes. I still see the red markers.

Reply:
it's hasn't been fixed yet I am helping Thomas he has issues pushing his code

Reply:
there you go done

Reply:
wait sorry I'm just rereading you comment I never marked it as resolved?